src/main/java/net/atos/zac/policy/PolicyService.java

```diff
@@ -145,6 +145,7 @@ public EnkelvoudigInformatieobjectActies readEnkelvoudigInformatieobjectActies(f
             enkelvoudigInformatieobjectData.zaakOpen = zaak.isOpen();
             enkelvoudigInformatieobjectData.zaaktype = ztcClientService.readZaaktype(zaak.getZaaktype()).getOmschrijving();
         }
+        enkelvoudigInformatieobjectData.ondertekend = enkelvoudigInformatieobject.getOndertekening().getDatum() != null;
```

Comment:
Is dit wel correct? Ik zou verwachten dat de ondertekening null is als ie niet ondertekend is?

Reply:
In de front-end code wel, maar in de back-end reflecteert de ondertekening de data in OpenZaak. Daar heeft elk EnkelvoudigInformatieObject een ondertekening, maar als er 'geen ondertekening' is zijn de datum en het type `null`. Waarom dat in OpenZaak zo geregeld is is mij ook onduidelijk, maar daar hebben we maar mee te leven.